src/lib/components/Swap/Summary/Summary.tsx

```diff
@@ -36,13 +39,13 @@ function TokenValue({ input, usdc, change }: TokenValueProps) {
       <Row gap={0.375} justify="flex-start">
         <TokenImg token={input.currency} />
         <ThemedText.Body2>
-          {input.toSignificant(6)} {input.currency.symbol}
+          {formatCurrencyAmount(input, 6, i18n.locale)} {input.currency.symbol}
         </ThemedText.Body2>
       </Row>
       {usdc && usdcAmount && (
         <Row justify="flex-start">
           <ThemedText.Caption color="secondary">
-            ${usdcAmount.toFixed(2)}
+            ${formatCurrencyAmount(usdcAmount, 2, i18n.locale)}
```

Comment:
If we're displaying this as USDC, should it be using US notation? Do we have anyone on the team knowledgable about financial i18n?

Reply:
bump

Reply:
https://uniswapteam.slack.com/archives/C028C31QRRV/p1644280687676229